### PR TITLE
Add organization and service_priority labels

### PR DESCRIPTION
**Pull request by QuentinBisson**
Adding missing labels from the agent metrics

## Checklist

I have:

- [x] Described why this change is being introduced
- [x] Separated out refactoring/reformatting in a dedicated PR
- [x] Updated changelog in `CHANGELOG.md`

### CODE REVIEW

File: service/controller/resource/monitoring/remotewriteapiendpointconfigsecret/resource.go [hunk 1]
```diff
@@ -127,9 +127,11 @@ func toSecret(ctx context.Context, v interface{}, config Config) (*corev1.Secret
 		key.ClusterTypeKey:  key.ClusterType(config.Installation, cluster),
 		key.CustomerKey:     config.Customer,
 		key.InstallationKey: config.Installation,
+		"organization":      key.GetOrganization(cluster),
```
Comment (marieroque) [inline service/controller/resource/monitoring/remotewriteapiendpointconfigsecret/resource.go:130]: Add a key.OrganizationKey ?

File: service/controller/resource/monitoring/remotewriteapiendpointconfigsecret/resource.go [hunk 2]
```diff
@@ -127,9 +127,11 @@ func toSecret(ctx context.Context, v interface{}, config Config) (*corev1.Secret
...
 		key.PipelineKey:     config.Pipeline,
 		key.ProviderKey:     config.Provider,
 		key.RegionKey:       config.Region,
+		"service_priority":  key.GetServicePriority(cluster),
```
Comment (marieroque) [inline service/controller/resource/monitoring/remotewriteapiendpointconfigsecret/resource.go:134]: Add a key.ServicePriorityKey ?

Comment (QuentinBisson) [reply service/controller/resource/monitoring/remotewriteapiendpointconfigsecret/resource.go]: Done

File: service/controller/resource/monitoring/remotewriteapiendpointconfigsecret/resource.go (hunk 1 above)
Comment (QuentinBisson) [reply service/controller/resource/monitoring/remotewriteapiendpointconfigsecret/resource.go]: Done

**Comment by marieroque** [comment]
Update CHANGELOG

**Comment by QuentinBisson** [comment]
I figured the changelog entry was not needed because it is technically already mentionned here https://github.com/giantswarm/prometheus-meta-operator/blob/master/CHANGELOG.md#added and not released yet

**Review by marieroque — APPROVED** [review]

**Pull request merged**